client/src/app/metadata-schema-form/custom/vf-input/vf-input.component.ts

```diff
@@ -67,7 +67,7 @@ export class VfInputComponent implements ControlValueAccessor, OnInit {
     'integer': 'number'
   };
 
-  onChange = (text: string) => {
+  onChange = (text: any) => {
```

Comment:
Why any?

Reply:
It was wrongly labelled before. I've renamed the var now but it can also be a boolean value since it's a checkbox. See line 126

Reply:
Maybe `string|boolean`?

Reply:
Well, it could potentially be null or undefined too. It's just we don't use it yet. This is a generic input component so should be generic

Reply:
ok